### PR TITLE
Mockserial

### PR DESCRIPTION
**Pull request by yozhgoor**
have made some change about to display the serial number in the menubar. WIP

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -181,6 +185,8 @@ function App() {
                 setNetwork={setNetwork}
                 ap={ap}
                 setAp={setAp}
+                serialNumber={serialNumber}
+                setSerialNumber={setSerialNumber}
```
Comment (cecton) [inline src/App.tsx]: You want to change the serial number from the menu?? :grin:

File: src/Menubar.css [hunk 2]
```diff
@@ -25,3 +25,10 @@
   display: block;
   width: 100%;
 }
+
+.bp3-text-small {
```
Comment (cecton) [inline src/Menubar.css]: Make a new class. Don't override existing blueprint classes

File: src/Menubar.tsx [hunk 3]
```diff
@@ -49,7 +49,12 @@ interface ApProps {
   setAp: (value: boolean) => void;
 }
 
-type MenubarProps = PhotoMode & Network & ConfigProps & ApProps;
+interface PortalProps {
+  serialNumber: string;
+  setSerialNumber: (value: string) => void;
```
Comment (cecton) [inline src/Menubar.tsx]: There is no reason for the menu to set the serial

File: src/Menubar.tsx [hunk 4]
```diff
@@ -147,6 +153,10 @@ const Settings = withContext(
           onClick={() => openPanel({ component: Advanced, title: "Advanced parameters" })}
         />
       </Menu>
+      <div className="bp3-text-small bp3-text-muted">
+        serial number : {serialNumber}
```
Comment (cecton) [inline src/Menubar.tsx]: In English we never add a space before: `:`, `?` and `!`

Comment (cecton) [reply src/Menubar.tsx]: Missing capital letter

File: src/Menubar.tsx [hunk 5]
```diff
@@ -147,6 +153,10 @@ const Settings = withContext(
...
+      </div>
+      </>
```
Comment (cecton) [inline src/Menubar.tsx]: Did you run `npm run fmt`??

**Comment by yozhgoor** [comment]
Some fix, thanks for the review

**Pull request closed without merging**